infiniboard-app/app/dashboard/dashboard.component.ts

```diff
@@ -67,10 +67,10 @@ export class DashboardComponent implements OnInit {
 
   private getWidgetComponentByType(widgetType: string): Type {
     switch (widgetType) {
-      case 'platform-status':
+      case  PlatformStatusWidgetComponent.WIDGETTYPE:
         return PlatformStatusWidgetComponent;
 
-      case 'jenkins-job':
+      case JenkinsJobWidgetComponent.WIDGETTYPE:
```

Comment:
I would prefer to Java code conventions for constants and use an underscore to seperate the words: `WIDGET_TYPE`.